_includes/setup/system-win.md

```diff
@@ -0,0 +1,11 @@
+## System requirements
+
+To install and run Flutter, your development environment must meet these minimum requirements:
+
+* **Operating Systems**: Windows (64-bit)
+* **Disk Space**: 400 MB (does not include disk space for Android Studio).
+* **Tools**: Flutter depends on these command-line tools being available in your environment.
+  * Git for Windows (with the Use Git from the Windows Command Prompt option), Cmd, PowerShell, `where`
```

Comment:
Maybe add a link to https://git-scm.com/download/win for git? It's the only thing listed here that doesn't ship with Windows by default.